extra-radio/qsstv/autobuild/beyond

```diff
@@ -1 +1 @@
-install -Dm644 "$SRCDIR"/qsstv/icons/qsstv.png "$PKGDIR"/usr/share/pixmaps/qsstv.png
+install -Dm644 "$SRCDIR"/icons/qsstv.png "$PKGDIR"/usr/share/pixmaps/qsstv.png
```

Comment:
Add verbosity, and `abinfo` here.

Reply:
I'd also suggest wrapping long lines with `\`, this makes your code easier to read. Personally I go no longer than 80 characters.

Reply:
Get it. Here I just simply corrected the path. Will fix soon.